metrics/prometheus/src/main/java/io/helidon/metrics/prometheus/PrometheusSupport.java

```diff
@@ -200,12 +208,13 @@ public static Builder builder() {
     /**
      * A builder of {@link PrometheusSupport}.
      */
-    public static final class Builder implements io.helidon.common.Builder<Builder, PrometheusSupport> {
+    public static final class Builder extends HelidonFeatureSupport.Builder<Builder, PrometheusSupport> {
 
         private CollectorRegistry registry = CollectorRegistry.defaultRegistry;
-        private String path;
+        private String path = DEFAULT_PATH;
 
         private Builder() {
+            super("/");
```

Comment:
I believe this should be `super(DEFAULT_PATH)`

Reply:
If I change this line to `super(DEFAULT_PATH)` the endpoint for metrics will be `...\metrics\metrics` but we need `...\metrics`

Reply:
roger that.